packages/utilities/psammead-styles/src/font-families.js

```diff
@@ -62,23 +70,320 @@ const helmetFontStyles = {
   `,
 };
 
-const nassimPersian = `font-family: NassimPersian, Arial, Verdana, Geneva, Helvetica, sans-serif;`;
+/*
+ *  BBC NASSIM
+ */
+
+const nassimArabicFontFamily = `font-family: "BBC Nassim Arabic", Arial, Verdana, Geneva, Helvetica, sans-serif;`;
+const nassimPashtoFontFamily = `font-family: "BBC Nassim Pashto", Arial, Verdana, Geneva, Helvetica, sans-serif;`;
+const nassimPersianFontFamily = `font-family: "BBC Nassim Persian", Arial, Verdana, Geneva, Helvetica, sans-serif;`;
+const nassimUrduFontFamily = `font-family: "BBC Nassim Urdu", Arial, Verdana, Geneva, Helvetica, sans-serif;`;
```

Comment:
Will we have to rename all these when/if we get Reith for Arabic-script sites?

Reply:
We'll delete all of the Nassim section in this file and create a ReithArabic section below the ReithLatin section. The exports at the bottom of the page for arabic, pashto, persian, urdu will be the same.